decentraland/explorer-desktop
# Fix: adds new profile HUD

## What does this PR change?

Fix #472
Adds the new profile HUD in desktop, based on the new webgl prefab
...

## How to test the changes?

1. Go to: https://renderer-artifacts.decentraland.org/desktop/index.html?dcl://DESKTOP-BRANCH=fix/new-profile-hud
2. Verify that in the profile edit panel everything works as in webgl

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
